pkg/occlient/occlient.go

```diff
@@ -2233,7 +2233,7 @@ func (c *Client) GetServiceClassesByCategory() (categories map[string][]scv1beta
 	categories = make(map[string][]scv1beta1.ClusterServiceClass)
 	classes, err := c.GetClusterServiceClasses()
 	if err != nil {
-		return nil, errors.Wrap(err, "unable to get cluster service classes")
+		return nil, err
```

Comment:
why is this required?

Reply:
I mean wrap also returns error just that the err is wrapped with a custom msg

Reply:
The problem is that LogErrorAndExit will unwrap all causes loses all context. We might need to revisit this function so that it's more useful.

Reply:
ok